python_modules/libraries/dagster-fivetran/dagster_fivetran/resources.py

```diff
@@ -1073,6 +1081,14 @@ def _sync_and_poll(self, context: AssetExecutionContext):
             connector_id=connector_id,
         )
 
+        # The FivetranOutput is None if the connector hasn't been synced
+        if not fivetran_output:
+            context.log.warning(
+                f"The connector with ID {connector_id} has not been synced."
```

Comment:
Might reword this as "The connector with ID x is currently paused and so it has not been synced. Make sure that your connector is enabled before syncing it with Dagster". or some iteration on that